### PR TITLE
Redesign of application

### PR DESCRIPTION
**Pull request by goldjay**
Screen Shot 2019-12-03 at 10 00 20 AM
Screen Shot 2019-12-03 at 10 00 29 AM
Screen Shot 2019-12-03 at 10 00 44 AM
Screen Shot 2019-12-03 at 10 00 53 AM
Screen Shot 2019-12-03 at 10 02 12 AM
Screen Shot 2019-12-03 at 10 02 42 AM

### CODE REVIEW

File: src/Components/Button.tsx [hunk 1]
```diff
@@ -35,17 +36,20 @@ class Button extends React.PureComponent<Props> {
...
       <button
-        className={`button_container ${enabledClassName} ${className || ""}`}
+        className={`button_container ${enabledClassName} ${className}`}
         disabled={disabled}
         name={name}
         type="button"
         onClick={this._onClick}
       >
+        {!!labelImg && (
+          <img className="button_image_label" alt={labelImg} src={labelImg} />
```
Comment (balderdash) [inline src/Components/Button.tsx]: `alt` should probably be `label`, not the same as src.

File: src/store/config.ts [hunk 2]
```diff
@@ -11,6 +13,9 @@ interface CustomPanelConfig extends TabConfig {
 
 export interface ActionConfig {
   label: string;
+  labelClassName?: string;
+  labelImg?: string;
+  postLabelElement?: JSX.Element;
```
Comment (balderdash) [inline src/store/config.ts]: Looks like this is unused.

Comment (goldjay) [reply src/store/config.ts]: It's used in the task panel to have the number of items to review colored.

Comment (balderdash) [reply src/store/config.ts]: That's as a prop to `LabelWrapper`, it doesn't also need to be in `ActionConfig` does it?

Comment (goldjay) [reply src/store/config.ts]: Gotcha. No, it doesn't need to be in ActionConfig.

**Pull request merged**